### PR TITLE
Improve Marker Popup Design!

### PR DESCRIPTION
**Pull request by sassypocoyo**
Finally made some long needed improvements to the Marker Popup design and I think it turned out pretty cool:

- Changed text font and position;
- Added some black gradient so the text stays visible despite the background;
- Added some icons to distinguish Date from Author;
- Made the author icon dynamic, so that pins with multiple authors will show a people icon;
- Also changed the hourglass icon to a clock icon on the Location list, adding the author icon too.

**Preview:**

image

### CODE REVIEW

File: components/Marker/Marker.tsx [hunk 1]
```diff
@@ -27,25 +35,28 @@ const Marker = ({
...
-            {date && (
-              <i>
-                {getFullDateString(date)} ({getRelativeTimeString(date)})
-              </i>
-            )}
-            <br />
-            <span>{name}</span>
+          <div className={styles.background}>
+            <div className={styles.text}>
+              <h1 style={{ lineHeight: '1.2' }}>
```
Comment (nelsonmestevao) [inline components/Marker/Marker.tsx:40]: Remove inline styles 🙏

File: components/Marker/Marker.tsx [hunk 2]
```diff
@@ -27,25 +35,28 @@ const Marker = ({
...
+                {city},<br />
+                {country}
+              </h1>
+              <span style={{ fontWeight: '400' }}>
```
Comment (nelsonmestevao) [inline components/Marker/Marker.tsx:44]: Same here 👆

File: components/Marker/Marker.tsx [hunk 3]
```diff
@@ -27,25 +35,28 @@ const Marker = ({
...
+              </span>
+            </div>
           </div>
           <Image
             alt={`${name} at ${city}`}
             src={photo}
             layout="fill"
             objectFit="cover"
-            style={{ borderRadius: '10px', overflow: 'hidden' }}
+            style={{ borderRadius: '10px', overflow: 'hidden', zIndex: '0' }}
```
Comment (nelsonmestevao) [inline components/Marker/Marker.tsx:59]: 👆

**Review by nelsonmestevao — DISMISSED** [review]

File: components/Marker/Marker.tsx (hunk 1 above)
Comment (sassypocoyo) [reply components/Marker/Marker.tsx]: Done ;) [1d15537](https://github.com/cesium/CeSIUMpeloMundo/pull/66/commits/1d15537b3e6bd83f4b2191df015cf9c427892442)

File: components/Marker/Marker.tsx (hunk 2 above)
Comment (sassypocoyo) [reply components/Marker/Marker.tsx]: Done ;) [1d15537](https://github.com/cesium/CeSIUMpeloMundo/pull/66/commits/1d15537b3e6bd83f4b2191df015cf9c427892442)

File: components/Marker/Marker.tsx (hunk 3 above)
Comment (sassypocoyo) [reply components/Marker/Marker.tsx]: And done ;) [1d15537](https://github.com/cesium/CeSIUMpeloMundo/pull/66/commits/1d15537b3e6bd83f4b2191df015cf9c427892442)

File: components/Marker/Marker.tsx [hunk 4]
```diff
@@ -27,25 +35,28 @@ const Marker = ({
...
-              <i>
-                {getFullDateString(date)} ({getRelativeTimeString(date)})
-              </i>
-            )}
-            <br />
-            <span>{name}</span>
+          <div className={styles.background}>
+            <div className={styles.text}>
+              <h1 className={styles.title}>
+                {city},<br />
```
Comment (nelsonmestevao) [inline components/Marker/Marker.tsx:41]: Maybe we don't need the <br/> anymore? What do you think?

Comment (sassypocoyo) [reply components/Marker/Marker.tsx:41]: You mean like:

`{city}, {country}`

image

?

Comment (nelsonmestevao) [reply components/Marker/Marker.tsx:41]: I think I prefer it. What do you like more?

Comment (sassypocoyo) [reply components/Marker/Marker.tsx:41]: I prefer it on most cases, but sometimes it looks like this:

image

Or this one: (now with the left spacing the same as the right)

image

I don't think it necessarily looks bad, just different. I think it's worth it, what do you think?

Comment (nelsonmestevao) [reply components/Marker/Marker.tsx:41]: I still prefer this way :+1:

Comment (sassypocoyo) [reply components/Marker/Marker.tsx]: It's done then :) :rocket:

**Review by nelsonmestevao — APPROVED** [review]

**Pull request merged**